redis/src/parser.rs

```diff
@@ -174,7 +186,7 @@ mod aio_support {
                 let buffer = &bytes[..];
                 let mut stream =
                     combine::easy::Stream(combine::stream::MaybePartialStream(buffer, !eof));
-                match combine::stream::decode_tokio(value(), &mut stream, &mut self.state) {
+                match combine::stream::decode_tokio(value(None), &mut stream, &mut self.state) {
```

Comment:
Just a question: I notice that there is a `decode_tokio()`. Is there another variant for other runtimes that we're not compiling in the default feature set/in CI?

Reply:
I'm not sure but it does look like `combine` may have an alternative decode macro for `async-std`.